### DISCUSSION TITLE
What's the equivalent of v5's `wavesurfer.container` in v7?

### DISCUSSION THREAD

**Discussion by kilgarenone** [katspaugh/wavesurfer.js#3354] [Q&A] [answered]
i'm upgrading wavesurfer from v5. I have codes doing the following:
```
let wavesurfer = WaveSurfer.create({ container: "#waveform"})

const { width, height, left, top } = wavesurfer.container.getBoundingClientRect()
```
After upgrading to v7, I got this error `TypeError: wavesurfer.container is undefined`. 

How would I get the 'container' in v7? 

Thanks!

**Comment by katspaugh** [comment, marked as answer]
You cannot get the internal container in v7, you should replace it with `document.querySelector("#waveform")`, i.e. your outer container.
